estate/models/buildings_model.py

```diff
@@ -0,0 +1,17 @@
+from odoo import models, fields   # noqa: F401
+
+class buidlings_model(models.Model):
+    _name = 'estate.buildings'
+    _description = 'Buildings Model'
+
+    name = fields.Char()
+    description = fields.Text()
+    value = fields.Integer()
+    garden_orientation = fields.Selection([
+        ('north', 'North'),
+        ('south', 'South'),
+        ('east', 'East'),
+        ('west', 'West')
+    ], string='Garden Orientation')
```

Comment:
Functionally, it would make more sense to have the string parameter as first argument. This will also allow you to remove its naming by simply writing the string itself like this:
```suggestion
    garden_orientation = fields.Selection(
        "Garden Orientation",
        selection=[
            ('north', 'North'),
            ('south', 'South'),
            ('east', 'East'),
            ('west', 'West'),
        ],
    )
```

Refer to the `Field` class for more information (https://github.com/odoo/odoo/blob/0a97fa64783e4b58bb17db79343fb566196e71eb/odoo/orm/fields.py#L92)